README.md

```diff
@@ -20,11 +20,16 @@ serverPort := uint16(8080)
 
 // prepare the server
 sPK, sSK := cipher.GenerateKeyPair()
+dmsgC := &DMSGClient{
```

Comment:
The capitalization of dmsg is confusing for me. Either we capitalize or do not. But that should be consistent. Both in comments and in code.

Reply:
It was initially done this way to mimic how Go handled [HTTP](https://golang.org/pkg/net/http/#HandlerFunc.ServeHTTP) and/or [TCP](https://golang.org/pkg/net/#DialTCP) (examples linked to each protocol).

Of course, I'm fine with either option, just giving some background on this...

Reply:
In dmsg repo, we use the lowercase spelling. So this should be used.